server/etc/regions.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "id": "f6db2e41-040e-47c7-8bba-a345b6d35ca1",
```

Comment:
https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/server/utils/premisesUtils.ts#L10C7-L10C15 isn't the same, it contains no user differentiation.

The API only has one consumer and its forked derivative. It's already differing its behaviour by requiring CAS1 and CAS3 flags on calls, so I don't see where the idea it shouldn't provide UI meta comes from. Besides, even if another consumer came along it would still need to know the user-group preference is to show the LA over the PDU.

If there are other id's for Greater Manchester they can be added to the regions.json. It's not the best but it beats behaviour changing on a visual string.

Reply:
> The delius code doesn't come back with the user region object.

We could propose that that happens. Though, Delius codes are still just strings set by Delius. 

We sadly don't have a universal ID for a region but perhaps we could set about making one.

> The API only has one consumer and its forked derivative. It's already differing its behaviour by requiring CAS1 and CAS3 flags on calls

As things stand there are three forked frontends as well as the integrations service which pulls information. There is no guarantee the architecture will remain this way though so we should be open to new consumers in future.

Reply:
I like the API design, it's almost like it's been designed to stop configuration leakage on purpose. 

Eventually the inevitable will happen here, the API, which is already CAS3 specific, will end up holding some CAS3 flags. Next week when a user says 'I used to be able to find a customer when I needed to (for example called Janet) but now I don't know what page they're on; I only see 10 records and have to click lots of times to find them. Please can I see 20 records instead?

Reply:
Here'a ticket from the board, it's PDU not LA, but you get the idea. I accept that time is pressing but someone has to maintain this service in the future and once you put this anti-pattern into play it'll be accepted and not challenged.

<img width="767" alt="Screenshot 2023-12-05 at 17 19 56" src="https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/7350883/7dcb7f97-060f-49a7-a5d0-6154527db092">

I've been doing this a long time Tom, I've seen it all before.

Reply:
The thing is by just adding one more column into that table all the problems disappear. You're thinking it's UI specific - it's not. A region manages properties by LA not PDU. This is a data attribute, and maybe it's required by the API itself, i.e when it generates reports? It not today, perhaps tomorrow - design in the flexibility from day 1.